src/Snapshot/Command/ScheduleSnapshot.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * This file is part of the broadway/snapshotting package.
+ *
+ *  (c) Qandidate.com <opensource@qandidate.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Broadway\Snapshotting\Snapshot\Command;
+
+class ScheduleSnapshot
+{
+    /**
+     * @var string
+     */
+    private $aggregateRootId;
+
+    /**
+     * @param string $aggregateRootId
+     */
+    public function __construct($aggregateRootId)
```

Comment:
I guess that would offer more flexibility. In the end it is up to the actual handler to decide in what fashion to handle the command.